src/main/java/com/lpvs/service/LPVSQueueService.java

```diff
@@ -271,7 +271,24 @@ public void processWebHook(LPVSQueue webhookConfig) {
             pullRequest = lpvsPullRequestRepository.saveAndFlush(pullRequest);
             log.error("Can't authorize commentResults() " + e);
             e.printStackTrace();
-            delete(webhookConfig);
+            int currentAttempts = webhookConfig.getAttempts();
+            if (currentAttempts < maxAttempts) {
+                webhookConfig.setAttempts(currentAttempts++);
+                try {
+                    addFirst(webhookConfig);
+                } catch (InterruptedException e1) {
+                    log.warn("Failed to update Queue element");
+                    e1.printStackTrace();
```

Comment:
We don't need to print stack trace. Please remove this line.
You can take just exception message for debug purpose.